bodhi/server/bugs.py

```diff
@@ -103,46 +92,46 @@ def _connect(self):
                                          cookiefile=None, tokenfile=None)
 
     @property
-    def bz(self):
+    def bz(self) -> bugzilla.Bugzilla:
         """
         Ensure we have connected to Bugzilla and return the client instance.
 
         Returns:
-            bugzilla.base.Bugzilla: A client Bugzilla instance.
+            A client Bugzilla instance.
         """
         if self._bz is None:
             self._connect()
         return self._bz
 
-    def get_url(self, bug_id):
+    def get_url(self, bug_id: typing.Union[int, str]) -> str:
         """
         Generate and return a URL to the given bug.
 
         Args:
-            bug_id (basestring or int): The id of the bug you want a URL for.
+            bug_id: The id of the bug you want a URL for.
         Returns:
             basestring: The requested URL.
         """
         return "%s/show_bug.cgi?id=%s" % (config['bz_baseurl'], bug_id)
 
-    def getbug(self, bug_id):
+    def getbug(self, bug_id: int) -> typing.Any:
```

Comment:
the bz method above returns a ``bugzilla.Bugzilla``, this one a ``bugzilla.bug.Bug`` according to the doc, any reason to specify ``bugzilla.bug.Bug`` here?

Reply:
Below we're specifying a type using the string (cf the ``models.Bug`` below), can it also be used in the returned type?